ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.test.js

```diff
@@ -690,4 +691,131 @@ describe('SnapUIRenderer', () => {
 
     expect(container).toMatchSnapshot();
   });
+
+  it('supports address inputs', () => {
```

Comment:
I wonder if we should move these component-specific tests to a `component.test.ts` 🤔

Reply:
yeah, it would be cleaner.

Reply:
In another PR we could move some of the code I wrote to setup the `SnapUIRenderer` to be shared and make it re-usable! Please create a ticket for this

Reply:
https://github.com/MetaMask/metamask-extension/issues/31292